### ISSUE TITLE
Jump to first interesting chapter - frontend

### ISSUE THREAD

**Issue by mulundapm** [bcc-code/bcc-media-app#392]
"Jump to  _name of chapter_" button floats on player for x seconds
First speech and panel unless its more than 50% of asset

x = duration on netflix

iOS and Android only. Feature Flag.

**Comment by andreasgangso** [comment]
Adopting the way netflix does it felt a bit unnatural. They have a specific "intro" or "previously" section to skip over, rather than a point to jump to. Which is a bit different since it makes sense for them to show the button for the entire intro (in the controls). If our "interesting chapter" is 40% in, its a bit weird to show that ugly button I made for the first 40%.

I didn't want to spend much time thinking about this so I did a pragmatic approach: It shows for 20 seconds / chapter start, but only while the controls is visible. The controls auto-hide after 5 seconds.

If you want me to polish it more let me know but imo its GEFN for a prototype
